values.go

```diff
@@ -1286,10 +1636,23 @@ type Capability struct {
 	BorrowType Type
 }
 
+func NewUnmeteredCapability(path Path, address Address, borrowType Type) Capability {
+	return Capability{
+		Path:       path,
+		Address:    address,
+		BorrowType: borrowType,
+	}
+}
+
+func NewCapability(gauge common.MemoryGauge, path Path, address Address, borrowType Type) Capability {
+	common.UseConstantMemory(gauge, common.MemoryKindCadenceCapability)
+	return NewUnmeteredCapability(path, address, borrowType)
+}
+
 func (Capability) isValue() {}
 
-func (Capability) Type() Type {
-	return CapabilityType{}
+func (v Capability) Type(gauge common.MemoryGauge) Type {
+	return NewCapabilityType(gauge, v.BorrowType) // TODO is this correct? original was nil for BorrowType
```

Comment:
I need advice here @turbolent.

Reply:
Yep, this makes sense 👍 It was previously not included